2 - RTB/Documentazione interna/Verbali/Esterni/23-11-23/23-11-23.typ

```diff
@@ -0,0 +1,77 @@
+#import "/template.typ": *
+
+//change title
+#show: project.with(
+  date: "23/11/23",
+  subTitle: "",
+  docType: "verbale",
+  externalParticipants : (
+    (name: "Matteo Bassani", role: "Referente aziendale"),
+  ), // Rimuovere se non ci sono partecipanti esterni
+  authors: (
+    "Zaccone Rosario",
+  ),
+  timeStart: "14:00",
+  timeEnd: "14:45",
+);
+
+= Ordine del giorno
+- Presentazione, supportata da una board Miro, delle assunzioni fatte dal team durante il periodo di individuazione degli use case;
+
+- Presentazione, supportata da una board Miro, delle domande raccolte dal team rigurdo l'analisi dei requisiti. Le domande si articolano in:
+ - use case;
+ - chiarimenti tecnici;
+ - device da supportare;
+ - requisiti non funzionali.
+
+- Richieste all'azienda proponente riguardo materiale utile durante il progetto;
+
+- Organizzazione di un prossimo meeting.
+\
+
+== Presentazione delle assunzioni fatte
+Il team ha iniziato il meeting presentando delle assunzioni al referente aziendale, per avere un riscontro riguardo la loro validità.\
+La prima assunzione specificava la presenza di un solo prodotto per ogni bin, ed è stata confermata e ritenuta una semplificazione utile. Per aree di carico/scarico questa assunzione non è valida, ma per semplicità di sviluppo si può tenere questa semplificazione.\
```

Comment:
Altra cosa che farei: trasformare questa lista in un elenco numerato (in Typst si crea come una lista puntata ma usando `+`)